### ISSUE TITLE
Very bad render/mesh - NeuS

### ISSUE THREAD

**Issue by hadillivvy** [Wi-sc/ViewFusion#1]
I tested the 3D reconstruction on the batman sample example provided and the render video and .ply look terrible. Any advice?

https://github.com/Wi-sc/ViewFusion/assets/150967838/c1d30edb-dd8e-44b2-b6fc-0b146f0f5d63

https://github.com/Wi-sc/ViewFusion/assets/150967838/65cde044-6c71-4294-a5e2-87827006193b

**Comment by Wi-sc** [comment]
Hi @hadillivvy ,

I just tested my code again, and it shouldn't be. Your problem is that the multi-view images are bad, not by the NeuS.

Can you provide more details? What model are you using? Have you checked the dataloader as the initial frame of the video seems broken?

**Comment by hadillivvy** [comment]
Hi @Wi-sc 

I don't think the issue is with the multi-view images because they look fine to me. Attached are a couple of them. To generate these, I used the `zero123-xl.ckpt` checkpoint and ran the generate_360_view_autoregressive.py script.

![input-0](https://github.com/Wi-sc/ViewFusion/assets/150967838/907561a4-4491-473f-8fa7-6508ffd8f7f4)
![input-7](https://github.com/Wi-sc/ViewFusion/assets/150967838/6f20f883-93f7-4a05-ab16-97af87210c97)
![input-20](https://github.com/Wi-sc/ViewFusion/assets/150967838/539f379b-bf9a-4dff-8d22-af2c8db1bc1a)
![input-26](https://github.com/Wi-sc/ViewFusion/assets/150967838/26c8bc2c-fcd1-473b-b6be-694d6daadc84)
![input-32](https://github.com/Wi-sc/ViewFusion/assets/150967838/35753193-10b7-4e4a-913a-2ef77a664d81)
![input-34](https://github.com/Wi-sc/ViewFusion/assets/150967838/7436fb73-6700-401a-8e49-fbee1015a7e2)

**Comment by Wi-sc** [comment]
Hey @hadillivvy ,

Thanks for the clarification. I have re-produced your issue on the batman image, while I have also run the pokeman images and the reconstruced 3d model looks well. So I think that is not a bug from the code.

I believe the issue here is because the batman is a thin and tall 3d object, with lots of details (cloak, belt, arms, masks, legs). We didn't focus on the NeuS training in our paper, and the training scripts and weights are stolen from SyncDreamer. Maybe you can try to reduce the weight of eikonal loss for NeuS and have a try. 

Sorry for the inconvenience. If you have any insights on this issue, just comment here. Let's make the model better!
